user-journey.adoc

```diff
@@ -75,25 +74,28 @@ sudo systemctl daemon-reload
 sudo systemctl restart docker
 ----
 .. Mac OSX
-... Configure a CentOS VM using as Hypervisor virtualbox or xhyve. The system will require at least 6 Gb of RAM and 4 CPUs - Process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a 
+... Configure a CentOS VM using as Hypervisor virtualbox or xhyve. The system will require at least 6 Gb of RAM and 4 CPUs - Process has been documented here: https://gist.github.com/cmoulliard/039e85e17ee99ad8c9e25b975d3f3a8a
 ... Next, the docker daemon and Red Hat certificates should be installed using the same instructions for CentOS/Fedora using yum or dnf (see 1. B., 3. a.).
-. Zip flow Java requirements
-.. http://www.oracle.com/technetwork/java/javase/downloads/jdk8-downloads-2133151.html[JDK 8}
-.. http://maven.apache.org/download.cgi[Maven 3.5]
-. Git flow client and account requirements
-.. https://git-scm.com/downloads[Git SCM client]
-.. User needs to have a github account, create dev credentials, and an API access token / key - see instructions https://github.com/fabric8-launcher/launcher-documentation/blob/master/docs/topics/proc_creating-a-github-personal-access-token.adoc[here].
+. *Determine the docker daemon `ETHERNET_IP_ADDRESS`*
++
+The ETHERNET_IP_ADDRESS, as defined within this doc, is the ethernet IP accessible from the host to the VM. This address could be part of a private network created behind the host and the guest = vm. It can also be the address of the docker daemon network adapter if running natively.
++
+TIP: If, the host address of your MacOS, Window or Linux desktop is “192.168.99.20” and the Centos Linux VM running the docker daemon is “192.168.99.50”, then, in this case, the ETHERNET_IP_ADDRESS is “192.168.99.50”
++
+Refer to https://dzone.com/articles/discovering-dockers-ip-address[Discovering docker daemon’s IP Address] if you need more guidance in determining the correct IP address in a native installation.
 
-== Use case scenario : “git flow”
-. User installs oc client and adds it to their path. (Note: a more recent version could be available from the *openshift-istio/origin* github repo! Make sure you have the latest `istiooc` version by going to https://github.com/openshift-istio/origin/releases  --- the following instructions assume version `*-alpha7`)
+== Use case scenario : `git flow`
+. Install istio `oc` client and adds it to your path.
```

Comment:
I think we want to format that as "istiooc" client, don't we?

Reply:
It's easy enough to update the docs when istiooc becomes oc.  In the meantime, we should document it as istiooc, because that's what the current client is.